File: 1796-second-largest-digit-in-a-string/1796-second-largest-digit-in-a-string.py
class Solution:
    def secondHighest(self, s: str) -> int:
        l=set()
        for i in s:
            if i.isdigit():
                l.add(int(i))
        a=sorted(l)
        if(len(a)<2):
            return -1
        return a[-2]

File: 1796-second-largest-digit-in-a-string/1796-second-largest-digit-in-a-string.cpp
class Solution {
public:
    int secondHighest(string s) {
        set<int>a;
        for(auto i:s){
            if(isdigit(i)){
                a.insert(i-'0');
            }
        }
        if(a.size()<2){
            return -1;
        }
        auto it=a.rbegin();
        it++;
        return *it;
        

    }
};